packages/accessibility/test/behaviors/testDefinitions.ts

```diff
@@ -167,11 +167,13 @@ function testMethodConditionallyAddAttribute(
   component,
   propertyDependsOn,
   valueOfProperty,
+  valueOfPropertyOtherwise,
   attributeToBeAdded,
   valueOfAttributeToBeAddedIfTrue,
   valueOfAttributeToBeAddedOtherwise,
 ) {
   const propertyWithAriaSelected = {}
+  propertyWithAriaSelected[propertyDependsOn] = valueOfPropertyOtherwise
```

Comment:
to reuse this method I had to prop the `otherwise` condition, which was by default `undefined`.